Dockerfile

```diff
@@ -151,60 +112,60 @@ FROM base AS tested
 
 COPY ./requirements-test.txt /tmp/requirements-test.txt
 
-RUN /opt/python/cp36-cp36m/bin/pip install -r /tmp/requirements-test.txt
-RUN /opt/python/cp37-cp37m/bin/pip install -r /tmp/requirements-test.txt
-RUN /opt/python/cp38-cp38/bin/pip install -r /tmp/requirements-test.txt
-RUN /opt/python/cp39-cp39/bin/pip install -r /tmp/requirements-test.txt
-RUN /opt/python/cp310-cp310/bin/pip install -r /tmp/requirements-test.txt
-RUN /opt/python/cp311-cp311/bin/pip install -r /tmp/requirements-test.txt
-RUN /opt/python/pp37-pypy37_pp73/bin/pip install -r /tmp/requirements-test.txt
-# RUN /opt/python/pp38-pypy38_pp73/bin/pip install -r /tmp/requirements-test.txt
+RUN /opt/python/cp37-cp37m/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt
+RUN /opt/python/cp38-cp38/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt
+RUN /opt/python/cp39-cp39/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt
+RUN /opt/python/cp310-cp310/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt
+RUN /opt/python/cp311-cp311/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt
+RUN /opt/python/pp37-pypy37_pp73/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt
```

Comment:
Right. Make you can change the base image to `aarch64` which doesn't mean it will immediately works)

I haven't find any way how to make any steps conditional. Basically this is not an aim for this pull request.

Reply:
I used bash if syntax, 
eg `RUN if [ "$PLAT" != "manylinux2014_aarch64" ] ; then /opt/python/pp37-pypy37_pp73/bin/pip install --only-binary pillow -r /tmp/requirements-test.txt ; fi`
which was effective, but yes, I guess it's not required for this pull.

Reply:
Also exclude tests for pypy on aarch64